cmd/insprd/memory/tree/manager.go

```diff
@@ -92,30 +91,36 @@ type PermTreeGetter struct {
 }
 
 // Apps returns a getter for apps on the root.
-func (t *PermTreeGetter) Apps() memory.AppGetInterface {
+func (t *PermTreeGetter) Apps() AppGetInterface {
```

Comment:
call this variable "ptg" instead of just "t", just to follow our pattern. The same for the functions below